--- guava/pom.xml	2012-02-22 14:59:24.000000000 +0100
+++ guava/pom.xml-gil	2012-04-28 13:55:54.793020732 +0200
@@ -84,7 +84,7 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
+      <!--plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>animal-sniffer-maven-plugin</artifactId>
         <version>1.7</version>
@@ -104,7 +104,7 @@
             </goals>
           </execution>
         </executions>
-      </plugin>
+      </plugin-->
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
